code_examples/sdk_examples/src/workshop/attester/generateDid.ts

```diff
@@ -1,51 +1,40 @@
 import { config as envConfig } from 'dotenv'
 
 import * as Kilt from '@kiltprotocol/sdk-js'
-
+// TODO: No longer needed in content, but needed for tutorial?
+// TODO: What to do?
+// TODO: And generateAccount needed changes, why?
 import { generateAccount } from './generateAccount'
```

Comment:
can't we use the generated account to create a DID?

Reply:
@rflechtner Yes, I guess I wanted to ensure it was "the right sort of account" to demonstrate the feature. I assume it is, my question here was also more around why we needed the changes here - https://github.com/KILTprotocol/docs/blob/1213e05ff9720775cac2c09fcfa246219ed71e72/code_examples/sdk_examples/src/workshop/attester/generateAccount.ts#L8

Reply:
That, I think, was due to how the input for the api.tx.did.* functions below are typed. The `type` key must be one of a set of known values, and somehow it was typed as `string` unless we added that.